core/mondoo-github-security.mql.yaml

```diff
@@ -5,6 +5,9 @@ policies:
     authors:
       - name: Mondoo, Inc
         email: hello@mondoo.com
+    tags:
+      mondoo.com/platform: github
```

Comment:
Do we want to start calling out `saas` as a platform? 🤔

Reply:
I was thinking about that too. Lets try